test/libyul/objectCompiler/nested_optimizer.yul

```diff
@@ -20,22 +20,26 @@ object "a" {
 // Assembly:
 //     /* "source":48:49   */
 //   0x00
-//   dup1
 //     /* "source":35:50   */
+//   dup1
 //   calldataload
 //     /* "source":107:127   */
 //   sstore
+//     /* "source":20:131   */
+//   stop
 // stop
```

Comment:
Why are there two stops here?

Reply:
The new code transform generates a stop at the end of the main block (instead of doing weirder things like jumping over functions), that's the new ``stop``. Not entirely sure where the one that was there before was coming from.